extensions/iam/decentralized-identity/identity-did-service/src/main/java/org/eclipse/dataspaceconnector/identity/DecentralizedIdentityService.java

```diff
@@ -88,7 +87,9 @@ public Result<ClaimToken> verifyJwtToken(TokenRepresentation tokenRepresentation
             PublicKeyWrapper publicKeyWrapper = KeyConverter.toPublicKeyWrapper(publicKeyJwk, publicKey.get().getId());
 
             monitor.debug("Verifying JWT with public key...");
-            if (!VerifiableCredentialFactory.verify(jwt, publicKeyWrapper)) {
+            Result<Void> verified = VerifiableCredentialFactory.verify(jwt, publicKeyWrapper, audience);
```

Comment:
plase use `var`

Reply:
In this pre-existing code block `var` is not used, and it was given as feedback in previous PRs to use `var` consistently or not to use it